q2_types/feature_data/tests/test_format.py

```diff
@@ -901,5 +903,37 @@ def test_blast6_format_invalid(self):
             BLAST6DirectoryFormat(temp_dir, mode='r').validate()
 
 
+class TestSequenceCharacteristicsFormat(TestPluginBase):
+    package = 'q2_types.feature_data.tests'
+
+    def test_sequence_characteristics_directory_format(self):
+        filepath = self.get_data_path('sequence_characteristics_length.txt')
```

Comment:
```suggestion
        filepath = self.get_data_path('sequence_characteristics_length.tsv')
```
We probably want to rename these files with the `.tsv` extension, for consistency.